postprocessing/janus-pp-rec.c

```diff
@@ -671,8 +671,8 @@ int main(int argc, char *argv[])
 		if(data) {
 			/* Things are simpler for data, no reordering is needed: start by the data time */
 			gint64 when = 0;
-			bytes = fread(&when, sizeof(gint64), 1, file);
-			if(bytes < (int)sizeof(gint64)) {
+			int elements_read = fread(&when, sizeof(gint64), 1, file);
+			if(elements_read < 1) {
```

Comment:
Wouldn't it be easier to just switch `sizeof(gint64)` and `1`? It would make it more consistent with our usage of `fread`, that AFAIK is all based on bytes and not on units.

Reply:
I agree.